core/src/main/java/org/springframework/security/access/AccessDeniedException.java

```diff
@@ -42,4 +42,12 @@ public AccessDeniedException(String msg, Throwable cause) {
 		super(msg, cause);
 	}
 
+	/**
+	 * @return The localized description of this <code>AccessDeniedException</code>.
```

Comment:
I would suggest adding a javadoc message before the `@return`, such as `Returns the localized description of this {@code AccessDeniedException}, by making use of a {@link SpringSecurityMessageSource} using the exception message as the key.`

You should also change `<code>` in `@return` to `{@code AccessDeniedException}` and remove the period at the end. Most returns messages start with a lower-case letter, so you may want to change "The" to "the" as well.